lib/basis_encode.cpp

```diff
@@ -397,7 +397,7 @@ static bool basisuEncoderInitialized = false;
  * @exception KTX_INVALID_OPERATION
  *                              Both preSwizzle and and inputSwizzle are specified
  *                              in @a params.
- * @exception KTX_OUT_OF_MEMORY Not enough memory to carry out supercompression.
+ * @exception KTX_OUT_OF_MEMORY Not enough memory to carry out compression.
```

Comment:
Please change the separateRGToRGB_A INVALID_OPERATION above. `s/is only 1D/has only 1 component/` and please delete "the texture's images are 1D" `KTX_INVALID_OPERATION` here and before `ktxtexture2_compressBasis`. There is no such check.

I'm asking for this here because other errors in this documentation are being fixed in this PR.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/8f42128b3c643db8b980d31167f8e9f34e1506e8